cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -176,8 +175,12 @@ <h2>URLs</h2>
             <h2>Data Selection</h2>
             {% elif record.type.primary == 'Environment' %}
             <h2>How was this built?</h2>
-            {% else %}
-            <h2>How were these data selected?</h2>
+            {% elif record.type.primary == 'Dataset' %}
+                {% if 'Simulated' in record.type.secondary %}
+                <h2>How were these data generated?</h2>
+                {% else %}
+                <h2>How were these data selected?</h2>
+                {% endif %}
```

Comment:
Beware, there is no final `else` clause that would display something for non-dataset types we might have.

Reply:
Good point. Also, for `Software` we have `Data Selection` as `<h2>` see [this record](http://opendata-dev.cern.ch/record/234). Should we change this?

Reply:
We could, although I am not sure what the good general heading title would be... At least in your concrete example, the record at hand says very early "Event selection" so it reads still nicely as it is.